app/models/account.rb

```diff
@@ -171,6 +171,10 @@ def domains
       reorder(nil).pluck('distinct accounts.domain')
     end
 
+    def inboxes
+      reorder(nil).where(protocol: :activitypub).pluck('distinct coalesce(nullif(accounts.shared_inbox_url, \'\'), accounts.inbox_url)')
```

Comment:
minor, but seems like using double quotes here would make it so you don't have to escape the sql single quotes?